scripts/combine/setupCombine.py

```diff
@@ -559,6 +559,7 @@ def setup(args, inputFile, fitvar, xnorm=False):
     cardTool.addSystematic("ecalL1Prefire", 
         processes=['MCnoQCD'],
         group="ecalPrefire",
+        splitGroup = {f"prefire" : f".*"},
```

Comment:
Ok, so electron pre-firing gets added to a common group with muon pre-firing, but the contribution is totally negligible?

Reply:
Both, muon and ecal prefiring are quite small, but I think it's better to combine it instead of not including it

Reply:
I think you should still define also muonPrefire group in splitGroup, otherwise you will only have "prefire". I think we need to keep that so that we can always check how much muonPrefire alone was (we know by now, but it can change).